### ISSUE TITLE
[demo] Use ms.nn.Normalization instead of torch.nn.BatchNorm1d

### ISSUE THREAD

**Issue by andafterall** [meta-soul/MetaSpore#92]
Use ms.nn.Normalization instead of torch.nn.BatchNorm1d: 
In parameter server architecture, we should use ms.nn.Normalization which handled global exponentially weighted moving average already.
